src/vscode-bicep/src/deploy/tree/AzLoginTreeItem.ts

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import * as semver from "semver";
+import { commands, Disposable, Extension, extensions } from "vscode";
+
+import {
+  AzExtParentTreeItem,
+  AzExtTreeItem,
+  GenericTreeItem,
+  IActionContext,
+  registerEvent,
+} from "@microsoft/vscode-azext-utils";
+
+import { localize } from "../../utils/localize";
+import { AzureAccount, AzureLoginStatus } from "../azure-account";
+import { GenericAzExtTreeItem } from "./GenericAzExtTreeItem";
+
+const signInLabel: string = localize("signInLabel", "Sign in to Azure...");
+const createAccountLabel: string = localize(
+  "createAccountLabel",
+  "Create a Free Azure Account..."
+);
+const signInCommandId = "azure-account.login";
+const createAccountCommandId = "azure-account.createAccount";
+const azureAccountExtensionId = "ms-vscode.azure-account";
+const extensionOpenCommand = "extension.open";
+
+type AzureAccountResult = AzureAccount | "notInstalled" | "needsUpdate";
+const minAccountExtensionVersion = "0.9.0";
+
+// Inspired from https://github.com/microsoft/vscode-azuretools/blob/main/azure/src/tree/AzureAccountTreeItemBase.ts,
+// without - list all subscription support
+export class AzLoginTreeItem extends AzExtParentTreeItem {
+  public static contextValue = "azureextensionui.azureAccount";
+  public readonly contextValue: string = AzLoginTreeItem.contextValue;
+  public readonly label: string = "Azure";
+  public childTypeLabel: string = localize("subscription", "subscription");
+  public autoSelectInTreeItemPicker = true;
+  public disposables: Disposable[] = [];
```

Comment:
`disposables` and `dispose()` can be removed.

Reply:
Removed here:
https://github.com/Azure/bicep/commit/11eccd63d7e635b0da47293d4761d62fa0b6593d

Thanks!